packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -78,6 +79,11 @@ export interface QuadraticGraphState extends InteractiveGraphStateCommon {
     coords: [Coord, Coord, Coord];
 }
 
+export interface SinusoidGraphState extends InteractiveGraphStateCommon {
+    type: "sinusoid";
+    coords: vec.Vector2[];
```

Comment:
Instead of an array, should `coords` be a pair of points `[vec.Vector2, vec.Vector2]`?